### DISCUSSION TITLE
Keeps Giving varied Responses

### DISCUSSION THREAD

**Discussion by Arndeverafter** [akki-io/laravel-google-analytics#27] [Q&A] [answered]
I have implemented this package in my project and Trying to access the data sometimes data points are missing which i believe i have found a workaround that but my concern is, comparing the results from the API and this https://ga-dev-tools.google/ga4/query-explorer/ , they are varied enough to make me think this is not a mistake. Especially the AverageSessionDurations & AverageSessionDurationByDate. eg the Package gives a json response of 7234.56388... while the playground only gives 1948.453838.... 

Any help would be appreciated.

**Comment by akki-io** [comment, marked as answer]
Sorry, but I am not able to reproduce this. The package does not manipulate any data and just returns what the query explorer returns.
